src/components/common/ColonyActions/helpers/mapItemToMessageFormat.tsx

```diff
@@ -8,16 +8,37 @@ import {
   ColonyAndExtensionsEvents,
   ColonyAction,
   ColonyActionType,
+  Domain,
 } from '~types';
 import { intl } from '~utils/intl';
 import { DEFAULT_TOKEN_DECIMALS } from '~constants';
 
+import { getDomainMetadataChangesValue } from './getDomainValues';
+
 import styles from './itemStyles.css';
 
 const { formatMessage } = intl({
   unknownDomain: 'UnknownDomain',
 });
 
+// Get contemporary domain name at the time of a transaction with a given hash (or fallback to the current name)
```

Comment:
I think I've never ever hear the term "contemporary" used in a codebase / coding environment :)

I like it :P

Reply:
Haha, I was struggling to find the exact word for it and I'm still unsure about its use in this context - but maybe our natives can correct me here

Reply:
Probably don't need both 'contemporary' and 'at the time of' since they're describing the same idea 🙂 I.e. "get domain name as it was at the time of a tx..."